tests/fov_watcher_test.py

```diff
@@ -29,24 +29,44 @@
 SLOW_COPY_INTERVAL_S = 1
 
 
-def _slow_copy_sample_tissue_data(dest: str, delta: int = 10, one_blank: bool = False):
+def _slow_copy_sample_tissue_data(
+    dest: str, delta: int = 10, one_blank: bool = False, temp_bin: bool = False
+):
     """slowly copies files from ./data/tissue/
 
     Args:
         dest (str):
             Where to copy tissue files to
         delta (int):
             Time (in seconds) between each file copy
+        one_blank (bool):
+            Add a blank .bin file or not
+        temp_bin (bool):
+            Use initial temp bin file paths or not
     """
 
     for tissue_file in sorted(os.listdir(COMBINED_DATA_PATH)):
         time.sleep(delta)
-        if one_blank and ".bin" in tissue_file:
+        if one_blank and ".bin" in tissue_file and tissue_file[0] != ".":
             # create blank (0 size) file
             open(os.path.join(dest, tissue_file), "w").close()
             one_blank = False
         else:
-            shutil.copy(os.path.join(COMBINED_DATA_PATH, tissue_file), dest)
+            tissue_path = os.path.join(COMBINED_DATA_PATH, tissue_file)
+            if temp_bin and ".bin" in tissue_file:
+                # copy to a temporary file with hash extension, then move to dest folder
+                new_tissue_path = os.path.join(COMBINED_DATA_PATH, "." + tissue_file + ".aBcDeF")
+                shutil.copy(tissue_path, new_tissue_path)
+                shutil.copy(new_tissue_path, dest)
+                os.remove(new_tissue_path)
+            else:
+                shutil.copy(tissue_path, dest)
+
+            # handle renaming
+            if temp_bin and ".bin" in tissue_file:
```

Comment:
These are the same conditions as the if statement above right? Could probably move this up instead of re-checking.

Reply:
Good idea!